aiozk/recipes/allocator.py

```diff
@@ -72,11 +72,13 @@ def validate(self, new_allocation):
         asyncio.ensure_future(self.monitor_member_changes())
 
     async def add(self, new_item):
-        new_set = self.full_set.copy().add(new_item)
```

Comment:
This should be in a separate commit with an appropriate test for the recipe. I understand that it is not working and why, but probably after fix it still isn't working.